src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -45,7 +45,7 @@ export class ReadingsGraphComponent implements OnDestroy {
   @ViewChild('assetChart') assetChart: Chart;
 
   public excludedReadingsList = [];
-  public assetReading = [];
+  public excludedReadingsSummaryList = [];
```

Comment:
`excludedReadingsSummaryList` should reset in toggleModal() and statsAssetReadingsGraph() methods.